paddle/fluid/operators/detection/prior_box_op.cc

```diff
@@ -149,6 +149,13 @@ class PriorBoxOpMaker : public framework::OpProtoAndCheckerMaker {
                    "(float) "
                    "Prior boxes center offset.")
         .SetDefault(0.5);
+    AddAttr<bool>(
+        "min_max_aspect_ratios_order",
+        "(bool) If set True, the output prior box is in order of"
+        "[min, max, aspect_ratios], which is consistent with Caffe."
```

Comment:
what if it's False?

Reply:
If it's False, this layer will output in the original order: [min, aspect_ratios, max]